src/dashboards/components/dashboard_index/DashboardCard.tsx

```diff
@@ -83,10 +93,11 @@ class DashboardCard extends PureComponent<Props> {
     )
   }
 
-  private handleUpdateDashboard = (name: string) => {
+  private handleUpdateDashboard = async (name: string) => {
     const {id, onUpdateDashboard} = this.props
 
     onUpdateDashboard(id, {name})
+    await updatePinnedItemByParam(id, {name})
```

Comment:
since i'm unfamiliar with pinnedItems, why is it important to await this?

Reply:
Do pinned items exist on the home page or are they pinned resources, like a dashboard pinned to a list of dashboards, a cell pinned to that dashboard, a panel pinned to a notebook, a notebook pinned to a notebook list?

Reply:
its on the homepage ill post a screenshot

Reply:
no worries. If that's the case do we need to make this awaited? Like, the user will still be on whatever resource, the promise should resolve when they get to the homepage / the home page will load up the resources when navigated to right? Seems minor, I don't know the implications of keeping this async or not but my impression of an async/await would have ramifications for the existing view and not some other part of the app that's not connected to the current state of affairs?

Reply:
thats a fair point, i was thinking of it from the perspective of whether or not the user would immediately expect it to show, and ramifications if it didn't show RIGHT away when they click on the page, but i guess thats both not an immediate concern and not something that outweighs the potential detriment to other experiences. Ill adjust this, thanks for changing my thinking on this!